Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -204,6 +213,16 @@ extension SPAppDelegate {
     func setupNoticeController() {
         NoticeController.shared.setupNoticeController()
     }
+
+    func performActionAfterUnlock(action: @escaping () -> Void) {
+        if isPresentingPasscodeLock && SPPinLockManager.shared.isEnabled {
+            verifyController?.addOnSuccesBlock {
```

Comment:
I know it's code just moved around... but should we unwrap first?

Otherwise this whole path will be a NO-OP, and the block will be lost?

Reply:
discussed, and decided that we are gonna disregard this one.  Thanks 🫡